src/main/java/io/jcoder/tutorials/ch11/CatchMultipleTypes/CatchMultipleTypesApp.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2018 - JCoder Ltd
+ */
+package io.jcoder.tutorials.ch11.CatchMultipleTypes;
```

Comment:
package names should be lowercased, can you rename to `io.jcoder.tutorials.ch11.multiplecatch` for example please? :)

Reply:
Fixed it and pushed the corrected version to my branch.